src/Storefront/Resources/config/packages/storefront.yaml

```diff
@@ -1,5 +1,4 @@
 storefront:
     theme:
-        file_delete_delay: 900
```

Comment:
does this need a deprecation?

Reply:
as answered in the other thread: IMHO no